### DISCUSSION TITLE
Docker set VFS Source Home Folder.

### DISCUSSION THREAD

**Discussion by bmartino1** [rejetto/hfs#792] [General]
Following an email response, I went down a journey to add HFS as a Unraid Docker in their community app store.
I'm currently working out some default bugs and configs...

https://github.com/bmartino1/hfs-docker/tree/main
as a fork form the offical docker repo:
https://github.com/damienzonly/hfs-docker

where there example:
 -v ./myDisk:/app/myDisk \
 
for some reason, I can't seem to get a variable to set this in the config at run. 
![image](https://github.com/user-attachments/assets/8a596131-e83a-4be5-8991-bb414b488d52)

per reading the potential options:
https://github.com/rejetto/hfs/blob/main/config.md#how-to-modify-configuration

I thought this would work...
HFS_VFS="[{'name': 'MyDisk', 'source': '/app/myDisk'}]"

I've tried HFS_vfs HFS_vfs_source
I'm not sure what the variable would be to set a home - root 
virtual file system disk source to path /app/myDisk

**Comment by bmartino1** [comment]
In HFS 2 this would be similar to the bind path to root option but in docker variable form.

This way in Unraid as a point click. I can set one path and ahve that accessible. User then can quickly access the admin interface and make self singed certs and pre defined a path beforehand to /app/mydisk as per the default docker run....

**Comment by rejetto** [comment]
sorry for the late reply.
you were quite close, the syntax is this: `HFS_VFS={"source": "/tmp"}`
json syntax requires double-quotes.
I never designed this usage, i.e. values that are not just numbers and text. So, it works almost accidentally.
I'm happy to see that your use-case seems feasible like it is, but I don't expect the same for other usages. I'm just not sure what's actually needed, so I'll wait to gather more requests on this.

**Reply by bmartino1** [reply]
awesome, that worked exactly what I need! Thank you
